msg/EscReport.msg

```diff
@@ -5,12 +5,14 @@ float32 esc_voltage					# Voltage measured from current ESC [V] - if supported
 float32 esc_current					# Current measured from current ESC [A] - if supported
 float32 esc_temperature					# Temperature measured from current ESC [degC] - if supported
 uint8 esc_address					# Address of current ESC (in most cases 1-8 / must be set by driver)
+uint8 esc_cmdcount					# Counter of number of commands
```

Comment:
Is this the size of the count from the actual ESC or should we make it larger?

Reply:
Yes, it is just an 8 bit field coming from the ESC. But perhaps we can make it larger future expansion.